composer.json

```diff
@@ -27,7 +27,7 @@
         "forum": "https://groups.google.com/forum/#!forum/twig-users"
     },
     "require": {
-        "php": ">=5.5",
+        "php": ">=5.6",
```

Comment:
I wouldn't trust PHP to be backwards compatible in the future, maybe use `5.6 || ^7.0`? You can always open it up for PHP 8 if its backwards compatible, but I wouldn't blindly trust it.